### PR TITLE
Adding aliases for getValue and setValue

### PR DESCRIPTION
**Pull request by scalvert**
Adding aliases for setValue/getValue.

Status: **Ready to merge**

Reviewers: @jansepar 

## Changes
- added `setChoice` and `getChoice` aliases

## How to test-drive this PR
- checkout branch
- `grunt test`

### CODE REVIEW

File: src/js/split-test.js [hunk 1]
```diff
@@ -61,13 +61,13 @@
...
      */
-    SplitTest.prototype.setValue = function(value) {
+    SplitTest.prototype.setChoice = SplitTest.prototype.setValue = function(value) {
         // Splits are stored for 30 days.
         SplitTest.setCookie(this.cookieName, value, this.cookieDomain);
     };
 
     /* SplitTest.getSplit returns the current split */
-    SplitTest.prototype.getValue = function() {
+    SplitTest.prototype.getChoice = SplitTest.prototype.getValue = function() {
```
Comment (jansepar) [inline src/js/split-test.js]: Comment could use an update.

File: src/js/split-test.js [hunk 2]
```diff
@@ -61,13 +61,13 @@
      * by calls to randomSplit() for example.
      *
      */
-    SplitTest.prototype.setValue = function(value) {
+    SplitTest.prototype.setChoice = SplitTest.prototype.setValue = function(value) {
```
Comment (jansepar) [inline src/js/split-test.js]: Comment here could also use an update.

File: bower.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "split-test",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (jansepar) [inline bower.json]: This isn't really a new feature release - should we just bump the patch instead of the minor?

Comment (scalvert) [inline bower.json]: Hmmm ya. I sorta was thinking it was, but I'm happy with a patch.

File: src/js/split-test.js (hunk 2 above)
Comment (scalvert) [inline src/js/split-test.js]: On it!

File: src/js/split-test.js (hunk 1 above)
Comment (scalvert) [inline src/js/split-test.js]: Ditto!

**Comment by jansepar** [comment]
:+1:

**Pull request merged**